commit 1c8d4944aeb2cfc19acb61d3b1813946b11edbdc (refs/changes/11/3111/4)
Author: Jared Morrow <jm@joyent.com>
Date:   2018-01-05T18:18:11+00:00 (1 year, 9 months ago)
    
    MANTA-3473 Update Muppet to node v4
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/targets.json.in b/targets.json.in
index fb6973c..5db07ce 100644
--- a/targets.json.in
+++ b/targets.json.in
@@ -1113,12 +1113,12 @@ cat <<EOF
     "build_platform": "20141030T081701Z",
     "image_name": "manta-loadbalancer",
     "image_description": "Manta loadbalancer",
-    "image_uuid": "de411e86-548d-11e4-a4b7-3bb60478632a",
+    "image_uuid": "04a48d7d-6bb5-4e83-8c3b-e60a99e0f48f",
     "pkgsrc": [
       "py27-curses",
-      "openssl-1.0.1u",
-      "stud-0.3p53nb2",
-      "zookeeper-client-3.4.6"
+      "openssl-1.0.2k",
+      "stud-0.3p53nb5",
+      "libzookeeper-3.4.6"
     ],
     "repos": [
       {"url": "git@github.com:joyent/muppet.git"}
